cli/flags/common.go

```diff
@@ -69,6 +69,13 @@ func (commonOpts *CommonOptions) InstallFlags(flags *pflag.FlagSet) {
 	flags.Var(opts.NewQuotedString(&tlsOptions.CertFile), "tlscert", "Path to TLS certificate file")
 	flags.Var(opts.NewQuotedString(&tlsOptions.KeyFile), "tlskey", "Path to TLS key file")
 
+	flags.String("namespace", "default", "Kubernetes namespace to use")
+	flags.SetAnnotation("namespace", "kubernetes", nil)
+	flags.SetAnnotation("namespace", "experimentalCLI", nil)
+	flags.String("kubeconfig", "", "Kubernetes config file")
+	flags.SetAnnotation("kubeconfig", "kubernetes", nil)
+	flags.SetAnnotation("kubeconfig", "experimentalCLI", nil)
```

Comment:
Why do these need to move to common? They aren't common to both orchestrators

Reply:
Well I mostly need `kubeconfig` (not `namespace`) to be common, or at least available, for `docker version` command, as you may specify different kubernetes configs and check the version against the pointing kubernetes.

Reply:
I see, but this flag isn't added to the `version` command, so how is it going to be read?